O/Octo/Compat.toml

```diff
@@ -1,5 +1,8 @@
 ["0.1"]
 julia = "0.7-1.3"
 
-["0.2"]
+["0.2.0-0.2.1"]
 julia = "1.0-1.3"
+
+["0.2.2"]
+julia = "1.0.3-1.3"
```

Comment:
Does this release not work on Julia 1.0?

Reply:
I just changed it to `julia = "1"`. thanks.